implementations/rust/ockam/ockam_api/src/nodes/models/address.rs

```diff
@@ -0,0 +1,19 @@
+use ockam_core::{Address, Route};
+use serde::Serializer;
+
+use crate::{addr_to_multiaddr, route_to_multiaddr};
+
+pub fn serialize_as_multiaddr<T, S>(t: &T, s: S) -> Result<S::Ok, S::Error>
+where
+    T: Into<Address> + Clone,
+    S: Serializer,
+{
+    s.serialize_some(&addr_to_multiaddr(t.clone()))
+}
+
+pub fn serialize_route_as_multiaddr<S>(route: &String, s: S) -> Result<S::Ok, S::Error>
+where
+S: Serializer,
+{
+	s.serialize_some(&Route::parse(route).and_then(|r| route_to_multiaddr(&r)))
+}
```

Comment:
Not ideal, but I'd move this to `utils.rs` where we have the rest of multiaddr utility functions. We'll probably move all those functions to its own file, but we can do that in a separate PR.

Reply:
Moved